### ISSUE TITLE
Is there any improvement on the view feature?

### ISSUE THREAD

**Issue by snapxtp** [EddyVerbruggen/nativescript-admob#29]
Hi;
is it possible to add Google ads to a scrollview ? I am sure it is one of the most needed things...
It would have been great if the ads were added to the scrollview areas, not the bottom or the top, in a fixed way.

**Comment by EddyVerbruggen** [comment]
Nope, but for anyone reading this: feel free to add a 👍or try creating a PR.

**Comment by bigradish** [comment]
+1
I think it will be better if the createBanner-like methods can just return a view (promise on success) containing the ad which can be put anywhere on the page. Or the view option can be set to a container view to display the ad.
BTW, on nativescript 3.4.3, I failed to show ads with this plugin.
